source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,75 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a query document that specifies the
+subset of your collection that you would like to retrieve. If you
+pass the ``FindOne()`` method an empty document, MongoDB will retrieve your
+entire collection. For more information on documents in the MongoDB Go driver,
+see our :doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify additional options for your ``FindOne()`` method call by passing
+the ``FindOne()`` method a ``*FindOneOptions`` value. For example, your
+``FindOneOptions`` value can specify a projection for your results, which
+configures the fields in the returned documents. Your ``FindOneOptions`` value
+can also specify a **sort**, which orders your results. 
+
+..
+   return + error
+
+The ``FindOne()`` method returns a value of type ``SingleResult``. The
+``SingleResult`` type represents a single document returned from a query to a
+MongoDB instance. You can call the ``SingleResult.Decode()`` method
+to unmarshall a ``SingleResult`` value into a variable with a type that can
+represent a document. If there was an error retrieving your document, that error
+will be returned by the ``Decode()`` method. If no documents matched your
+query, ``Decode()`` will return an ``ErrNoDocuments`` error. 
+
+Example
+-------
+
+The following code example retrieves a single document from the ``movies``
+collection. It uses the following functions and methods:
+
+- An empty **context** returned from the ``context.TODO()`` function.
+
+- A **configuration** that specifies a **sort** on matched documents in
+  descending order by rating, such that the returned document has the highest
+  rating. Our configuration also specifies a **projection** for our results so
+  that only the ``title`` and ``imdb`` fields are retrieved.
+
+- A **query document** that is passed to the ``FindOne()`` method. The query
+  only matches movies with the title exactly matching the text ``'The Room'``.
```

Comment:
Sorry, this is not a standard that I am proposing. I did this because the equivalent "The Room" uses single quotes in both the [Node Docs](https://github.com/mongodb/docs-node/blob/0f80a7850bb34085818e8c1458dd7f1849e46e1c/source/usage-examples/findOne.txt#L39) and the [Java Docs](https://github.com/mongodb/docs-java/blob/b4828b8b01ef02794e38bbe478c19eaa1cc70cb2/source/usage-examples/findOne.txt#L40). Would you like these changed as well?

FIx for go docs:  https://github.com/mongodb/docs-golang/pull/2/commits/5e983a731739ebdc9dd2c083cdf8464f04f10a91

Reply:
Feel free to use whatever and we can discuss as a team. I'll refrain from making any comments about identifying fields and values for now.